tests/UI/campaigns/functional/BO/03_catalog/04_attributesAndFeatures/01_attributes/06_changeAttributePosition.js

```diff
@@ -0,0 +1,181 @@
+require('module-alias/register');
+
+// Helpers to open and close browser
+const helper = require('@utils/helpers');
+
+// Common tests login BO
+const loginCommon = require('@commonTests/loginBO');
+
+// Import pages
+const dashboardPage = require('@pages/BO/dashboard');
+const attributesPage = require('@pages/BO/catalog/attributes');
+const viewAttributePage = require('@pages/BO/catalog/attributes/view');
+
+// Import data
+const {Attributes} = require('@data/demo/attributes');
+
+// Import test context
+const testContext = require('@utils/testContext');
+
+const baseContext = 'functional_BO_shopParameters_catalog_attributesAndFeatures_attributes_changeAttributePosition';
+
+// Import expect from chai
+const {expect} = require('chai');
+
+// Browser and tab
+let browserContext;
+let page;
+
```

Comment:
No comment to explain the test ?

Reply:
The describe say all what needed to know, no need for further explaining

Reply:
Description added to the test